frontend/src/components/common/SectionFilterHeader.tsx

```diff
@@ -41,6 +45,11 @@ export default function SectionFilterHeader(props: SectionFilterHeaderProps) {
   });
 
   React.useEffect(() => {
+    const namespace = getFilterValueByNameFromURL('namespace', location);
+    if (namespace.length > 0) {
+      dispatch(setNamespaceFilter(namespace));
```

Comment:
Before we dispatch, we should check whether the current namespace is already the same value as the URL, to avoid reacting to this change by other components.

The fundamental idea is that redux is the one source of truth, the URL is simply the entry point when loading the page and for copying the URL (and from that moment, should be kept up to date with the redux value).